CodeReview/Amie_code.py

```diff
@@ -21,14 +24,22 @@ def f(data):
     data['Minute'] = pd.DatetimeIndex(data['Date Time']).minute
     data['Time'] =  data['Hour'] + data['Minute']/60
     return data
+
 data1 = f(data)
+#Only grab 2014 data
 data2014 = data1.query('Year == 2014')
+#Only grab January data
 data012014 = data2014.query('Month == 1')
+#Only grab January first data
 data01012014 = data012014.query('Day == 1')
+#Edit data si it only has time and water depth data
```

Comment:
You should be able to use the original data frame instead of creating a new one.